### ISSUE TITLE
Build fails when used as dependency

### ISSUE THREAD

**Issue by uneco** [cloudflare/quiche#154]
The error that occurs:
```
error[E0423]: expected function, found macro `log::set_boxed_logger`
  --> /Users/x/.cargo/registry/src/github.com-1ecc6299db9ec823/quiche-0.1.0-alpha4/src/ffi.rs:66:5
   |
66 |     log::set_boxed_logger(logger).unwrap();
   |     ^^^^^^^^^^^^^^^^^^^^^ help: use `!` to invoke the macro: `log::set_boxed_logger!`
```

Steps to reproduce:
```shell
mkdir quiche-test
cd quiche-test
cargo init
cargo add quiche@0.1.0-alpha4 // adding `quiche = "0.1.0-alpha4"` to Cargo.toml
cargo build
```

The solution I found, add "std" feature to Cargo.toml (on dependent side, not a quiche):
```toml
[dependencies]
log  = { version = "0.4.8", features = ["std"] }
quiche = "0.1.0-alpha4"
```

**Comment by junhochoi** [comment]
Looks like similar issue in https://github.com/rust-lang-nursery/log/issues/54

What version of rust are you using? Can you try again with latest rustc release?

**Comment by uneco** [comment]
I'm already using the latest version of rustc, but there was a problem.

```
info: syncing channel updates for 'stable-x86_64-apple-darwin'
info: checking for self-updates
info: downloading self-update

  stable-x86_64-apple-darwin unchanged - rustc 1.37.0 (eae3437df 2019-08-13)
```

Strangely, this error does not occur when cloning `cloudflare/quiche` repository and building.

**Comment by uneco** [comment]
![unknown (1)](https://user-images.githubusercontent.com/603523/64650083-45d0cb00-d459-11e9-9b13-3251efc1b31d.png)

During the investigation, I confirmed that the extra-filename included in the built deps differ only for `log` and `liblog`.
In case of failure, only `liblog-33c22466c4f7b128.rlib` is included, whereas in case of success,` liblog-645e5fdeeb84cf7d.rlib` is included.

**Comment by ghedo** [comment]
Given that using quiche from git solves the problem, I'm guessing the issue has already been fixed at some point in the past since the last proper reelase. So I'm closing this (though feel free to reopen if you disagree).

We'll probably cut another release in the next 1-2 weeks.
